app/views/dashboard/_user_collections.html.erb

```diff
@@ -1,60 +1,24 @@
 <div id="collection" class="dashboard_card-view active">
   <div class="dashboard_card-interface_action-bar">
+    <div class="dashboard_card-interface_view-toggles">
+      <div class= "dashboard_card-interface_toggle-view__btn" onclick="Helpers.toggleElementById('collection-grid', 'collection__togglable-view')">
```

Comment:
Nice to have: An active style so you know which of the two buttons is "selected." Would be toggled here, I suppose.

Reply:
Great idea, I actually have a toggleActive function I wrote last night.

Reply:
Pushed up this fix, and boooooyy howdy, I need to fix all these class names, ohh lawwdd they long